language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -113,13 +117,13 @@ class DottyLanguageServer extends LanguageServer
         drivers(config)
       case None =>
         val config = drivers.keys.head
-        println(s"No configuration contains $uri as a source file, arbitrarily choosing ${config.id}")
+        // println(s"No configuration contains $uri as a source file, arbitrarily choosing ${config.id}")
         drivers(config)
     }
   }
 
   override def connect(client: LanguageClient): Unit = {
-    this.client = client
+    myClient = client.asInstanceOf[WorksheetClient]
```

Comment:
You can avoid the asInstanceOf by changing `connect` to not be an override but just a regular method that takes a `LanguageClient` (the override comes from extending `LanguageClientAware` but this is not actually useful for anything).